app/models/account.rb

```diff
@@ -6,4 +6,5 @@ class Account < ApplicationRecord
   validates :services, presence: true
   validates :owner_request, inclusion: { in: [true, false] }
   validates :account_managers, presence: true
+  validates :domain, format: { with: %r{\A(http:|https:)//[^.]+\..*\z} }
```

Comment:
eddig nem volt kötelező domaint megadni, jó lenne, ha ezután is üresen lehetne hagyni ezt a mezőt